core/src/main/java/com/sequenceiq/cloudbreak/converter/stack/cluster/ClusterRequestToClusterConverter.java

```diff
@@ -64,7 +64,7 @@ public Cluster convert(ClusterRequest source) {
         FileSystemBase fileSystem = source.getFileSystem();
         convertAttributes(source, cluster);
         if (fileSystem != null) {
-            cluster.setFileSystem(fileSystemConfigService.getPrivateFileSystem(fileSystem.getName(), authenticatedUserService.getCbUser()));
+            cluster.setFileSystem(fileSystemConfigService.getByNameForOrganizationId(fileSystem.getName(), restRequestThreadLocalService.getRequestedOrgId()));
```

Comment:
not working on V1 API calls. if ```restRequestThreadLocalService.getRequestedOrgId()``` returns null, use the default org. although @mhmxs's next PR going to change this behaviour.

Reply:
I asked him to do on this way.